vnda/hooks/useCart.ts

```diff
@@ -47,6 +47,7 @@ const state = {
   update: action("vnda/actions/cart/updateCart.ts"),
   addItem: action("vnda/actions/cart/addItem.ts"),
   updateItem: action("vnda/actions/cart/updateItem.ts"),
+  simulate: invoke.vnda.actions.cart.simulation,
```

Comment:
shouldn't this be wrapped as well?
```suggestion
  simulate: action("vnda/actions/cart/simulation"),
```